modules/installation-special-config-mirrored-disk.adoc

```diff
@@ -87,3 +87,73 @@ $ podman run --pull=always --rm --volume $HOME/clusterconfig:/pwd --workdir /pwd
 This command produces a combined Ignition config in `$HOME/clusterconfig/worker-raid.ign`.
 +
 . Continue with the remainder of the {product-title} deployment, using the combined Ignition config to provision nodes.
+
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume.
+
+. Create a machine config that configures a data volume by using software RAID.
+
+* To configure a data volume with RAID 1 on secondary disks, save the following machine config sample to a file in the `<installation_directory>/openshift` directory. For example, name the file `raid1-alt-storage.yaml`:
++
+.RAID 1 on secondary disks
+[source, yaml]
+----
```

Comment:
Rather than making this a bullet, I'd make it a continuation (on a new line) of the step it's under. A lot of times in our docs, we do something like this:
```suggestion
. Create a machine config that configures a data volume by using software RAID in the `<installation_directory>/openshift` directory.
+
.Sample RAID 1 on secondary disks configuration file named `raid1-alt-storage.yaml`
[source, yaml]
----
```
(My example file .header could probably stand some improvement, but just trying to illustrate :slightly_smiling_face: You can see lots of examples if you search `.Sample` in the repo.)